### PR TITLE
Updated on teacher excluded

### PR DESCRIPTION
**Pull request by heliumonoxide**

### CODE REVIEW

File: mainFV2.cpp [hunk 1]
```diff
@@ -122,34 +123,34 @@ int main(int argc, char** argv)
...
+        std::cout << "how many human: " << humans << std::endl;
+        std::cout << "delay: " << delay << std::endl;
+        std::cout << "dt: " << clock() - now << std::endl;
+        std::cout << "capacity: " << humans*100/full_capacity-1 << "%" << std::endl;
+
+        //Delay send
+        if (clock() - now > delay){
+            std::cout << "Delayed print" << std::endl;
+            ofstream MyFile("humans.txt");
+            MyFile << humans - 1;
```
Comment (chronomustard) [inline mainFV2.cpp]: change to if humans > 0, humans - 1 for output

Comment (chronomustard) [reply mainFV2.cpp]: or capacity + 1 for full capacity

**Review by chronomustard — CHANGES_REQUESTED** [review]

File: mainFV2.cpp (hunk 1 above)
Comment (heliumonoxide) [reply mainFV2.cpp]: Noted, Thanks

**Review by chronomustard — APPROVED** [review]
approved

**Pull request merged**